cli/azd/pkg/project/internal/project_utils.go

```diff
@@ -34,3 +37,25 @@ func CreateDeployableZip(appName string, path string) (string, error) {
 
 	return zipFile.Name(), nil
 }
+
+const c_gitIgnore string = ".gitignore"
+
+// CreateSkipPatternsFromGitIgnore inspect root project path and a `servicePath`
+// to see if there is a .gitignore file. It then combine both files in a single list
+// of exclusions.
+func CreateSkipPatternsFromGitIgnore(servicePath string) ([]gitignore.GitIgnore, error) {
```

Comment:
My preference would be that this was just an unexported function in the project package.  It is only called from that package.  I dislike how the code looks with `internal` infront of a bunch of calls.

Reply:
I was following the current approach in code.
I have removed the `internal` package in the interest of unexported functions